static/src/stylesheets/module/atoms/_snippets.scss

```diff
@@ -0,0 +1,66 @@
+.explainer-snippet__image {
+    float: left;
+    margin-right: $gs-gutter / 2;
+
+    display: inline-block;
+    position: relative;
+    width: 33%;
+    height: 0;
+    overflow: hidden;
+    border-radius: 50%;
+    padding-top: 33%;
+}
+
+.explainer-snippet__image img {
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    top: 0;
+    left: 0;
+    object-fit: cover;
+}
+
+.explainer-snippet__item::before {
+    content: '';
+    display: flex;
+    width: 200px;
+    @include mq($until: mobileLandscape) {
```

Comment:
Could you move this down a little bit (after the last property) for readability?